ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth63.java

```diff
@@ -40,7 +41,7 @@
 
     private static final EthVersion version = V63;
 
-    @Autowired
+    @Autowired @Qualifier("defaultRepository")
```

Comment:
Nope, that wouldn't work. `defaultRepository` is just a bean for initializing `BlockchainImpl` (probably not the best solution). You may do `BlockchainImpl.getRepository()` here